controllers/clientset_test.go

```diff
@@ -186,9 +186,7 @@ var _ = Describe("Disruption Client", func() {
 				// Assert
 				Eventually(func() watch.Event {
 					event := <-watcher.ResultChan()
-					log.Infow("Event received", "Type", event.Type, "Object", event.Object)
 					if event.Type == eventType {
-						log.Infow("Event matched", "Type", event.Type, "Expected", eventType, "Object", event.Object)
```

Comment:
I'd rather have this in here? We could tune them down to `log.Debugw` instead of `log.Infow`? It would make it much easier to debug a failure in the future